cmd/lotus-storage-miner/market.go

```diff
@@ -40,29 +48,115 @@ var disableCmd = &cli.Command{
 	},
 }
 
-var setPriceCmd = &cli.Command{
-	Name:  "set-price",
-	Usage: "Set price that miner will accept storage deals at (FIL / GiB / Epoch)",
-	Flags: []cli.Flag{},
+var setAskCmd = &cli.Command{
+	Name:  "set-ask",
+	Usage: "Configure the miner's ask",
+	Flags: []cli.Flag{
+		&cli.Uint64Flag{
+			Name:     "price",
+			Usage:    "Set the price of the ask (specified as FIL / GiB / Epoch) to `PRICE`",
+			Required: true,
+		},
+		&cli.Uint64Flag{
+			Name:        "duration",
+			Usage:       "Set the duration (specified as epochs) of the ask to `DURATION`",
+			DefaultText: "100000",
+			Value:       100000,
+		},
+		&cli.StringFlag{
+			Name:        "min-piece-size",
+			Usage:       "Set minimum piece size (without bit-padding, in bytes) in ask to `SIZE`",
```

Comment:
Since we are parsing those with `RAMInBytes`, Those should probably be with bit-padding, otherwise it's going to be very annoying

Reply:
It would be very annoying. Good point. Updated to use bit-padded quantities.